common/djangoapps/track/segment.py

```diff
@@ -11,15 +11,31 @@
 from urlparse import urlunsplit
 
 import analytics
+from crum import get_current_request
 from django.conf import settings
 from eventtracking import tracker
+from track import contexts
+from track.views import _get_request_value
 
 
-def track(user_id, event_name, properties=None, context=None):
+def track(user_id, event_name, properties=None, context=None, send_to_track=False, user=None):
```

Comment:
nit: it will save the line below `properties = properties or {}`
```suggestion
def track(user_id, event_name, properties={}, context=None, send_to_track=False, user=None):
```

Reply:
If someone sends `None` or any falsie intentionally, the code will break then.

Reply:
If we are not using the variables sent to the function, those must not be in parameters? what do you say?

Reply:
didn't get that.

Reply:
leave it.